### DISCUSSION TITLE
Problem with Playstation Plus Titlebar

### DISCUSSION THREAD

**Discussion by Armin1979** [r57zone/DualShock4-emulator#66] [Q&A]
Hi! 

I think your Emulator is almost perfect for use with the PS Plus application, but I have one problem when using keyboard+mouse: Always when moving the mouse the title bar of the PlayStation Plus application appears and when stop moving the mouse it disappears again. This is unfortunately quite annoying. Is there any way I can solve this somehow with the current PS Plus app?

**Comment by r57zone** [comment]
Hi, use the hotkeys to press full screen mode ALT" + "F10"

**Reply by r57zone** [reply]
You may need to change the program title/caption in the configuration file, maybe

**Reply by Armin1979** [reply]
Okay thanks, I actually tried Alt+F10 already before, but now, when using it multiple times, it works. Thanks for your quick answer!

**Reply by Armin1979** [reply]
btw is it possible to use multiple keys (keyboard assignments) for one controller button?

**Reply by r57zone** [reply]
At the moment this is not possible
